.github/config/master.yaml

```diff
@@ -53,7 +53,7 @@ flavors:
         cache_repository: "build"
         organization: "quay.io/costoolkit"
         skip_tests: false
-        run_tests: ["test-fallback"]
+        run_tests: ["test-smoke", "test-upgrades-images-signed", "test-upgrades-images-unsigned", "test-upgrades-local" ]
```

Comment:
maybe we can just keep the unsigned tests for now? I guess we want to test if the upgrade scenario works, It should be enough x86_64 covering the signed scenarios, no?

Reply:
yes definetly. at the times we didn't had elemental tests at all, so that was our only way of coverage. if elemental-cli have unsigned/signed tests we can cut here just with unsigned to have an integration test